actions/trigger-argo-workflow/README.md

```diff
@@ -0,0 +1,49 @@
+# Trigger Argo Workflow Action
+
+This GitHub action triggers an Argo workflow in one of the Grafana Labs Argo
+Workflows instances. It contains a small wrapper around the `argo` CLI, which is
+downloaded as part of the action.
+
+As our Argo Workflows instances require authentication, this workflow will only
+work when triggered from a `grafana`-owned repository. We would welcome
+contributions to extend this action to work with any instance, or you are free
+to fork and modify to run on your own instances.
+
+## How to use
+
+## Inputs
+
+- `instance`: The instance to use (`dev` or `ops`). Defaults to `ops`.
+- `namespace`: Required. The namespace to trigger the workflow in.
+- `command`: The command to run. Defaults to `submit`.
```

Comment:
What do you want me to do? I wanted to support people running anything they want, but if you want I can remove that.

Reply:
I think our 2 short-term options are:
* make at least the action singularly scoped to submit workflows; then it would match the readme description and name of this whole action
* update the name and description of the action to reflect that this action is generic and able to run any argo workflows CLI command. right now, the description, `This GitHub action triggers an Argo workflow in one of the Grafana Labs Argo
Workflows instances.` and name, `trigger-argo-workflow`, suggest to me that the action is only able to submit workflows.

I think both are valid options. Slightly leaning to just update the description and name since you've already configured the wrapper tool to run any arbitrary command. I just think the way we're describing the tool isn't reflective of what it can do.

Reply:
32f367468e20e737743f91ad6890f383f322f34c